ndc-lambda-sdk/package.json

```diff
@@ -30,7 +30,7 @@
     "url": "git+https://github.com/hasura/ndc-nodejs-lambda.git"
   },
   "dependencies": {
-    "@hasura/ndc-sdk-typescript": "^1.2.8",
+    "@hasura/ndc-sdk-typescript": "^3.0.0",
```

Comment:
Big jump! 👍

Reply:
Well, technically this PR went to v2.0.0 first, but then I added the code to go to v3.0.0. 🤷‍♂️